src/main/scala/com/gu/octopusthrift/aws/CloudWatch.scala

```diff
@@ -0,0 +1,49 @@
+package com.gu.octopusthrift.aws
+
+import com.amazonaws.services.cloudwatch.AmazonCloudWatch
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchClientBuilder
+import com.amazonaws.services.cloudwatch.model.Dimension
+import com.amazonaws.services.cloudwatch.model.MetricDatum
+import com.amazonaws.services.cloudwatch.model.PutMetricDataRequest
+import com.amazonaws.services.cloudwatch.model.PutMetricDataResult
+import com.amazonaws.services.cloudwatch.model.StandardUnit
+import com.gu.octopusthrift.Config
+import com.gu.octopusthrift.services.Logging;
+
+object Metrics {
+  val MissingComposerIdOrBodyText = "MissingComposerIdOrBodyText"
```

Comment:
It would involved some refactoring, but it might be useful to see the split between these two. Alternatively, if we want to keep this quite generic, we could call it something like `MissingMandatoryData`?

Reply:
I did consider this but was put off by the idea because the other metrics are mutually exclusive whereas missing body text or composer id can occur simultaneously. I was concerned this would effect how the data was interpreted. I like the idea of renaming it and would consider splitting out the metrics if you think those concerns are unfounded.

Reply:
I don't think either change is necessarily essential. I'm probably just thinking too far ahead, to if/when we start to consume something like the headline articles too and potentially want to write similar-ish metrics for missing data that doesn't include Composer ID or the Body Text object type.

We can always just iterate if/when we get to that point 👍